frontend/src/Dashboard/index.js

```diff
@@ -17,77 +17,83 @@ export const Dashboard = props => {
   const [activePage, changePage] = useState(1);
   const [classesOnPage, setCurrentClasses] = useState([]);
 
-  const fetchClasses = () => {
+  const fetchClasses = useCallback(() => {
```

Comment:
This doesn't look right

Reply:
Yeah the linter didn't know what you are trying to do. You can check that out and it should explain to you why it's throwing errors.